src/shared/error-boundary/ErrorBoundary.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { Component, ErrorInfo, ReactNode } from 'react';
+import { Link } from 'react-router-dom';
+
+import { postError } from '_mocks_';
+
+import { Modal, Button } from 'ui';
+
+export namespace ErrorBoundary {
+  export interface Props {
+    children: ReactNode;
+  }
+
+  export interface Error {
+    name: string;
+    message: string;
+    componentStack: string;
+    occuredAt: Date;
+  }
+
+  export interface State {
+    hasError: boolean;
```

Comment:
errors state property is not needed - errors is used only to track errors not for display on UI - so should be added as class property not state property